README.md

```diff
@@ -25,11 +25,10 @@ python3 -m pip install --user git+https://github.com/systemd/mkosi.git
 If you want to hack on mkosi do
 ```shell
 # clone either this repository or your fork of it
-git clone https://github.com/systemd/mkosi/
-cd mkosi
-python3 -m pip install --user --no-use-pep517 --editable .
+git clone https://github.com/systemd/mkosi/ <PROJECT_PATH>
+[sudo|pkexec] <PROJECT_PATH>/bin/mkosi
 ```
-This will install mkosi in editable mode to `~/.local/bin/mkosi`, allowing you
+This will run mkosi as present in the module source, allowing you
```

Comment:
"module source" is python jargon. Maybe just say "This will run mkosi from this checkout, …" ?